pwa-devdocs/scripts/markdown-linter/tokenizers/jekyll-link.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * A remark-parse tokenizer to add support for jekyll style link definitions 
+ * These definitions have spaces in the link and are not correctly parsed
+ *
+ * For more information see:
+ * - https://github.com/remarkjs/remark/tree/master/packages/remark-parse#extending-the-parser
+ */
+function jekyllLinkDefinition() {
+    var Parser = this.Parser;
+    var tokenizers = Parser.prototype.inlineTokenizers;
+    var methods = Parser.prototype.inlineMethods;
```

Comment:
If possible, we'd prefer to use `const` and `let` instead of `var` throughout this file.